Restore.cmd

```diff
@@ -1,38 +1,61 @@
 @echo off
 @setlocal
 
-set NuGetExe="%~dp0NuGet.exe"
-set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%~dp0nuget.config" -Project2ProjectTimeOut 1200
+set RoslynRoot=%~dp0
+set NuGetExe="%RoslynRoot%\NuGet.exe"
+set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%RoslynRoot%\nuget.config" -Project2ProjectTimeOut 1200
+
+:ParseArguments
+if /I "%1" == "/?" goto :Usage
+if /I "%1" == "/clean" set RestoreClean=true&&shift&& goto :ParseArguments
+goto :DoneParsing
+
+call :Usage && exit /b 1
```

Comment:
I think that is just boiler plate I copied from another script.  Will remove here in the next change that does the actual downloading.